packages/model-viewer/src/three-components/ModelUtils.ts

```diff
@@ -49,17 +74,21 @@ export const reduceVertices = <T>(
             for (i = 0, l = vertices.length; i < l; i++) {
               vertex.copy(vertices[i]);
               vertex.applyMatrix4(object.matrixWorld);
-
               value = func(value, vertex);
             }
 
           } else if (geometry.isBufferGeometry) {
             const {position} = geometry.attributes;
 
             if (position !== undefined) {
+              const scale = getNormalizedComponentScale(position);
+
               for (i = 0, l = position.count; i < l; i++) {
-                vertex.fromBufferAttribute(position, i)
-                    .applyMatrix4(object.matrixWorld);
+                vertex.fromBufferAttribute(position, i);
+                if (position.normalized) {
```

Comment:
Is this check necessary? Won't scale always be 1 if position is not normalized? Or could we make it that way?

Reply:
This was cribbed from Three's update but we return 1 to cover the Int32 case, Three seems to ignore this case but we are a bit more strict because of typescript. Also this depends on the flag being set _and_ the buffer type being Int16 (for instance) but it _should_ be common that you have a Int16 buffer that is _not_ normalized so we definitely need the check somewhere.